api/src/lib.rs

```diff
@@ -135,5 +135,8 @@ pub async fn run(pool: PgConnectionPool) {
 
     // run it with hyper on localhost:3000
     let listener = tokio::net::TcpListener::bind("0.0.0.0:3000").await.unwrap();
-    axum::serve(listener, app).await.unwrap();
+    axum::serve(listener, app)
+        .with_graceful_shutdown(::util::shutdown_signal())
```

Comment:
The leading scoping operator before `util` looks a little odd here, is it correct?

Reply:
The leading scoping operator needs to be there for the code to compile.

Reply:
Done!